doc/ref_arch/openstack/chapters/chapter04.md

```diff
@@ -115,58 +115,36 @@ Networks nodes are mainly used for L3 traffic management for overlay tenant netw
 
 #### 4.2.2.5. Compute Nodes
 
-This section specifies the compute node configurations to support the flavors. Because a flavor has the same capabilities but different geometry sizes, the term “flavor series” is used.
+This section specifies the compute node configurations to support the Basic and High Performance profiles; in OpenStack this would be accomplished by creating "flavors". In certain cases, different nodes (hosts, physical servers) in the OpenStack cloud infrastructure may be configured with a profile but with a different geometry (such as CPU over-suscription). The cloud operator may choose to create a "flavor series" as a set of standard configurations, of a given profile, capturing some of this variability.
```

Comment:
@pgoyal01 
Rather than "flavor series" can we name it "profile series" in order to keep "profiles" for host and flavor for VM?

Reply:
@karinesevilla Thanks. I think "profile extensions" capture what was the intent originally.

Reply:
Yes okay, resolved